core/lib/tasks/core.rake

```diff
@@ -32,9 +39,9 @@ use rake db:load_file[/absolute/path/to/sample/filename.rb]'
 
   desc "Migrate schema to version 0 and back up again. WARNING: Destroys all data in tables!!"
   task remigrate: :environment do
-    require 'highline/import'
+    ActiveSupport::Deprecation.warn("remigrate has been deprecated. Please demigrate and then migrate again, I guess?")
```

Comment:
I think `rake db:reset` is what we want to recommend, no?

Also `Spree::Deprecation.warn`